tools/extras/install_miniconda.sh

```diff
@@ -0,0 +1,14 @@
+ #!/bin/bash
+
+# Choose default settings for installation
+# Miniconda should be installed in the HOME directory. This is the default setting.
+# Please don't change it.
+# Also don't make miniconda's python as default.
+
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
```

Comment:
try to make this so that if someone just runs it, any necessary instructions will be printed to the screen, and also the script will exit with a suitable error message if some stage of this procedure fails.

The other installation scripts are much more automated: you can just run them and they install the thing.